doc/source/raysgd/v2/api.rst

```diff
@@ -8,93 +8,93 @@ RaySGD API
 Trainer
```

Comment:
Can you add a comment at the top of this file mentioning this symlink? Since the API references other files in the v2 package.